clojure_quickstart.org

```diff
@@ -0,0 +1,53 @@
+* Clojure REPL quickstart
+
+There are a few options on how you can start playing with Clojure.
+Choose one that suits you.
+
+** Install Leiningen
+
+Leiningen is what npm is for JavaScript. You can install it in a couple of
+ways:
+
+*** From repository:
```

Comment:
I would recommend adding a note for all the `apt` commands that they a related to the Debian-based distributions (like Ubuntu).